CONTRIBUTING.md

```diff
@@ -0,0 +1,172 @@
+
+In Next.js, this is done by adding `transpilePackages` in the `next.config.mjs` files. In Tsup builds, we simply tell it
+to bundle everything into a single file.
+
+Examples of both can be found in the different applications in the `apps/` directory. For example, `apps/web` uses
+Next.js, and `apps/gateway-email` uses TSup.
+</details>
+
+## Tools
+
+The following tools are used to develop Monoweb:
+
+- [Node.js](https://nodejs.org/): JavaScript runtime
+- [PNPM](https://pnpm.io/): Package manager (`npm i -g pnpm`)
+- [Docker](https://www.docker.com/): Containerization for a few applications (not required `apps/web`)
+- [Docker Compose](https://docs.docker.com/compose/): For using local PostgreSQL if wanted (modern versions of Docker
+  include Docker Compose)
+- [Doppler](https://doppler.com/): Secrets management, used for retrieving secrets in development.
+    - If you are not a Dotkom team member, you will not have access to Doppler. You can still run the applications, but
+      you will have to provide the secrets yourself. In this case, you can use a `.env` file in the different
+      application directories, or export them into your shell.
+- [AWS CLIv2](https://aws.amazon.com/cli/): Used for deploying to applications to AWS. You will need to have the correct
+  AWS credentials set up to deploy to AWS. If you are not a dotkom team member, we can unfortunately not provide you
+  with credentials.
+- [Terraform](https://www.terraform.io/): Infrastructure as code, used for managing the infrastructure. Note that the
+  Terraform configuration is located in a separate, private GitHub repository. If you are not a dotkom team member, we
+  can unfortunately not provide you with access to the repository. This is done to prevent any confidential information
+  from being leaked through [Atlantis](https://www.runatlantis.io/).
+
+## Local Development
+
+To get started with local development, ensure you have the applicable tools installed. To build and run all the
+applications, you can use the following commands:
+
+```bash
+doppler login
+doppler setup --project monoweb --config dev
+
+git clone https://github.com/dotkom/monoweb
+cd monoweb
+
+pnpm install
+doppler run pnpm dev
+```
+
+> Note that this, by default, uses a shared database hosted on https://neon.tech. Because this database is shared, you
+> should be careful with what you do with it. If you need to perform migrations or change data, you should set up your
+> branch on Neon. See the [guide for using Neon like a pro](./using-neon-like-a-pro.md) for more information.
+
+If you want to run a specific application, you can use the `--filter` flag:
+
+```bash
+doppler run pnpm dev --filter=@dotkomonline/web
+```
+
+If you have some local variables that you want to use, and override the Doppler ones you can use the `--preserve-env`
+flag:
+
+```bash
+export DATABASE_URL="postgres://<username>:<password>@localhost:5432/<db_name>"
+doppler run --preserve-env pnpm dev
+```
+
```

Comment:
```suggestion

If you are not using Doppler, you need to use a standalone `.env` file placed in the project root. For how to populate the `.env` file, see [this chapter](#required-environment-variables). For how to start local development, follow the rest of the chapter where you omit the use of Doppler:

```diff
- doppler run pnpm dev
+ pnpm dev
#```

```
I suggest we document how to run without Doppler in addition to the rest.

_(Remove the `#` in front of the closing code block)_